lib/services/local/access_list.go

```diff
@@ -302,6 +321,154 @@ func (a *AccessListService) AccessRequestPromote(_ context.Context, _ *accesslis
 	return nil, trace.NotImplemented("AccessRequestPromote should not be called")
 }
 
+// ListAccessListReviews will list access list reviews for a particular access list.
+func (a *AccessListService) ListAccessListReviews(ctx context.Context, accessList string, pageSize int, pageToken string) (reviews []*accesslist.Review, nextToken string, err error) {
+	err = a.service.RunWhileLocked(ctx, lockName(accessList), accessListLockTTL, func(ctx context.Context, _ backend.Backend) error {
+		_, err := a.service.GetResource(ctx, accessList)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		reviews, nextToken, err = a.reviewService.WithPrefix(accessList).ListResources(ctx, pageSize, pageToken)
+		return trace.Wrap(err)
+	})
+	if err != nil {
+		return nil, "", trace.Wrap(err)
+	}
+	return reviews, nextToken, nil
+}
+
+// CreateAccessListReview will create a new review for an access list.
+func (a *AccessListService) CreateAccessListReview(ctx context.Context, review *accesslist.Review) (createdReview *accesslist.Review, err error) {
+	reviewName := strings.ToLower(uuid.New().String())
```

Comment:
```suggestion
	reviewName := uuid.New().String()
```
nit: uuid string always returns lowercase hex